### PR TITLE
fix: 🐛 missing tags

### PR DESCRIPTION
**Pull request by gertzakis**
Closes: #164 Fix missing Tags from some forms and serializers.

### CODE REVIEW

**Review by chadell — COMMENTED** [review]
LGTM

File: nautobot_bgp_models/tables.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ class BGPRoutingInstanceTable(StatusTableMixin, BaseTable):
     device = tables.LinkColumn()
     autonomous_system = tables.LinkColumn()
     router_id = tables.LinkColumn()
-    tags = TagColumn(url_name="plugins:nautobot_bgp_models:BGPRoutingInstance_list")
+    tags = TagColumn(url_name="plugins:nautobot_bgp_models:bgproutinginstance_list")
```
Comment (glennmatthews) [inline nautobot_bgp_models/tables.py:46]: Do we need to add a `tags` column to PeerGroupTable and PeerEndpointTable as well? Looks like maybe so?

**Review by glennmatthews — APPROVED** [review]

File: nautobot_bgp_models/tables.py (hunk 1 above)
Comment (gertzakis) [reply nautobot_bgp_models/tables.py:46]: Nice catch! Thank you

**Review by glennmatthews — APPROVED** [review]

**Review by mzbroch — APPROVED** [review]

**Comment by glennmatthews** [comment]
Let's make sure this fix gets into `develop` as well!

**Comment by mzbroch** [comment]
> Let's make sure this fix gets into `develop` as well!

Thanks - adding via #177

**Pull request merged**